newswires/client/package.json

```diff
@@ -48,23 +48,22 @@
 		"@types/sanitize-html": "^2.13.0",
 		"@vitejs/plugin-react": "^4.3.1",
 		"eslint": "^8.57.0",
-		"eslint-import-resolver-typescript": "^3.6.3",
-		"eslint-plugin-prettier": "^5.2.1",
+		"eslint-import-resolver-typescript": "^4.2.2",
+		"eslint-plugin-prettier": "^5.2.4",
 		"eslint-plugin-react": "^7.35.0",
-		"eslint-plugin-react-hooks": "^5.1.0-rc.0",
-		"eslint-plugin-react-refresh": "^0.4.9",
+		"eslint-plugin-react-hooks": "^5.2.0",
+		"eslint-plugin-react-refresh": "^0.4.19",
 		"eslint-plugin-storybook": "0.11.3",
-		"globals": "^15.9.0",
+		"globals": "^16.0.0",
 		"jest": "29.7.0",
 		"jest-environment-jsdom": "^29.7.0",
-		"prettier": "^3.3.3",
+		"prettier": "^3.5.3",
 		"storybook": "8.6.0",
 		"timezone-mock": "^1.3.6",
-		"ts-jest": "29.2.4",
+		"ts-jest": "29.2.6",
 		"typescript": "^5.5.3",
-		"typescript-eslint": "^8.0.1",
-		"vite": "^5.4.1",
```

Comment:
`vite` is being moved to the root workspace in this branch, but I can't see it being used outside of the `client` workspace?

Reply:
I had to move it to fix the following error: https://github.com/guardian/newswires/actions/runs/14089633373/job/39462772723#step:10:10